lang/main_pl.arb

```diff
@@ -112,27 +112,27 @@
     "copied_to_clipboard": "Skopiowano do schowka",
 
     "translate": "Przetłumacz",
-    "random": "Losowy",
-    "custom": "Własny",
-    "enter_text_here": "Wprowadź tutaj tekst",
+    "random": "Losowa",
+    "custom": "Własna",
+    "enter_text_here": "Wpisz tutaj tekst",
     "check": "Sprawdź",
 
     "something_went_wrong": "Coś poszło nie tak",
-    "check_instance": "Sprawdź instancję",
+    "check_instance": "Sprawdź instancję.",
     "no_internet": "Brak połączenia z Internetem",
     "ok": "OK",
 
-    "settings": "Settings",
+    "settings": "Ustawienia",
 
-    "select_instance": "Select instance",
-    "theme": "Theme",
-    "light": "Light",
-    "dark": "Dark",
-    "follow_system": "Follow system",
+    "select_instance": "Wybierz instancję",
+    "theme": "Motyw",
+    "light": "Jasny",
+    "dark": "Ciemny",
+    "follow_system": "Zgodny z systemowym",
 
-    "about": "About",
-    "about_description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc molestie, tellus id tincidunt faucibus, odio ante tristique metus, id tincidunt dui mauris vel est.",
+    "about": "O aplikacji",
+    "about_description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc molestie, tellus id tincidunt faucibus, odio ante tristique metus, id tincidunt dui mauris vel est."
 
-    "not_available": "Not available"
+    "not_available": "Niedostępne"
```

Comment:
Wouldn't it be better "Niedostępny" when it comes to a dictionary. But that's just my suggestion.

Reply:
I was thinking about it as a  "(Tłumaczenie) niedostępne".

Reply:
But this message appears when selecting "Tłumacz" and not when translating. If the translator appeared and his window had this message, it could be labeled "Niedostępne" :wink:.

Reply:
I know where is it, no need for a screenshot 🙂 

Think of it in another way - there is just one word, and everybody can add any noun to it. We as translators have to decide what should it be. If there would be anything more than "not available" (what would be better) there would be just one version. For now, we can just argue which version is better. One (like me) can think of "Tłumaczenie niedostępne", "LibreTranslate niedostępne"*. Others (like you) may think of "Tłumacz niedostępny", "LibreTranslate niedostępny"*. Both are correct and one has to be chosen. I'm for "niedostępne", but you can convice me otherwise 🙂

*English own names can be treated with any grammatical gender if not used with any additional noun, see here: <https://sjp.pwn.pl/poradnia/haslo/rodzaj-gramatyczny-nazw-wlasnych;5448.html> (example with Apple is the same as ours with LibreTranslate)

Reply:
> but you can convice me otherwise

I will try 😁.

Often, translators use neuter words because they do not know what the text string is referring to. Only a good translation is good when the translator reviews the code or properly analyzes the application in terms of translation and functions. Not all do it. In this case, it does not matter, because there are not many text strings in this application.

@ManeraKai  I'm sorry, but now I will write in Polish so that we can communicate better with Aga-C in our mother tongue 😉.

Teraz przeanalizujemy działanie użytkownika.

1. Uruchamia aplikację.
2. Wybiera rodzaj tłumacza np. "LibreTranslate".
3. Ukazuje się mu informacja "Niedostępne".

I teraz co wybrał? Tłumaczenie, rodzaj tłumaczenia, czy rodzaj tłumacza? Moim zdaniem w jego myślach jest, że wybrał rodzaj tłumacza w osobie męskiej, to dlaczego dostał komunikat w rodzaju nijakim? Zaczyna się zastanawiać, co jest nie dostępne. Przecież wybrał tłumacza, a nie tłumaczenie. Tłumaczenie następuje dopiero w następnych krokach, czyli po wpisaniu tekstu do tłumaczenia i naciśnięciu przycisku "Przetłumacz".

Drugi przykład. Załużmy, że użytkownik nie zna jeszcze dobrze aplikacji. Uruchamia ją, przechodzi do drugiego tłumacza i widzi komunikat "Niedostępne". Zaczyna się zastanawiać, co jest niedostępne. Patrzy, co wybrał i widzi "LibreTranslate", czyli rodzaj tłumacza, który powinien być niedostępny, to dlaczego mam komunikat "Niedostępne". I kwituje to jednym słowem: Ach Ci tłumacze, znowu coś "poknocili".

Przekonałem, czy jeszcze nie?

Reply:
Ale może też wziąć pod uwagę, że są **niedostępne** funkcje. Dobra, pozostawiamy tak jak jest 😉.

Reply:
Tylko widzisz, z drugiej strony użytkownik widzi na ekranie LibreTranslate, więc sobie nic więcej nie dopowiada, tylko czyta "LibreTranslate niedostępne" 🙂 Szczerze, już sama nie wiem, jak jest lepiej. Najprościej by było, gdyby w oryginale było pełne zdanie np. "Service is not available" i wtedy tłumaczymy z jedną, konkretną odmianą.

@ManeraKai How about you'd extend a bit the "Not available" text to make it a full sentence, like "Service is not available" or some other way. It'll still be short and it will be easier to translate to other languages.

Reply:
@Aga-C  I thought about it too, to expand the message into a sentence 😉.